python-requirements.in

```diff
@@ -61,7 +61,7 @@ types-tabulate==0.8.11
 anytree==2.8.0
 
 # Development version with OT-specific changes
-https://github.com/lowRISC/fusesoc/archive/refs/tags/ot-0.4.zip
+https://github.com/lowRISC/fusesoc/archive/refs/tags/ot-0.5.zip
```

Comment:
Might need to run this to update python-requirements.txt too:
```
./util/sh/scripts/gen-python-requirements.sh
```

Reply:
Not might, must :)

Reply:
Indeed, just done that and pushed the updated commit